### PR TITLE
WEBUI-995: add functional test to capture the records in activity tab

### PR DESCRIPTION
**Pull request by Pranit-Sotre**

### CODE REVIEW

File: packages/nuxeo-web-ui-ftest/features/step_definitions/ui.js [hunk 1]
```diff
@@ -18,3 +18,9 @@ Then('I can see {string} in the Activity feed', function(activity) {
   this.ui.activityFeed.waitForVisible();
   this.ui.activityFeed.getActivity(activity).waitForVisible().should.be.true;
 });
+Then('I click the blob download button', function() {
+  const page = this.ui.browser.documentPage(this.doc.type);
+  page.waitForVisible();
```
Comment (Nisha-QA) [inline packages/nuxeo-web-ui-ftest/features/step_definitions/ui.js]: Why we need to wait for this when we have another wait in the next line?

File: ftest/features/activity.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Feature: Create Picture and validate activity feed
+
+  Background:
+    Given I login as "Administrator"
+    And I have a Workspace document
+    And I have a File document
+    And I browse to the document
+
+  Scenario: Validate activity tab entries
+    When I click the Create Document button
```
Comment (Nisha-QA) [inline ftest/features/activity.feature:9]: why do we need to create a document here if we already have a file document in the background?

**Review by Nisha-QA — APPROVED** [review]
Looks good

File: ftest/features/activity.feature (hunk 2 above)
Comment (Pranit-Sotre) [reply ftest/features/activity.feature:9]: If we create new document then we have all the entries from start, then it is effective to validate all entries.
That's why here we need to create a document.

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by Nishant0928 — APPROVED** [review]

**Review by satishyadav-dev — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**